src/ducks/transactions/TransactionModal.styl

```diff
@@ -1,6 +1,6 @@
 .TransactionModalHeading
     text-align center
-    padding-top 3px
+    padding-top 4px
```

Comment:
rem are better than pixels because they make your design responsive and respects the zoom level of the user : https://blog.lesieur.name/pourquoi-j-utilise-l-unite-rem-et-non-l-unite-pixel/

4px -> .25rem

Is there a particular reason not to use them here ?